content/services/cryoem.rst

```diff
@@ -0,0 +1,11 @@
+CryoEM
+=======
+:slug: services/powerdev/cyroem
+:author: Maximillian Schmidt
+:menu: POWERLinux/OpenPOWER Development Hosting, CryoEM, 6
+
+The Open Source Lab (OSUOSL) and Center for Quantitative Life Sciences (CQLS) partner with IBM and OpenPOWER in order to provide a download for a new package: CryoEM. Compiled for PPC64LE, this is one thing research can achieve on OpenPOWER.
```

Comment:
I'm not so sure about the grammar for the last sentence or the context it provides.

Reply:
I figure Chris or Ken will come back with something more official for a page. I just left it vague and up to interpretation.

Reply:
You might have them take a quick look at this and provide feedback. I realize this is just an initial page.